execution-plane/arcon/Cargo.toml

```diff
@@ -24,13 +24,18 @@ arcon_macros = { version = "0.1", path = "arcon_macros" }
 arcon_error = { version = "0.1", path = "arcon_error" }
 arcon_extra = { version = "0.1", path = "arcon_extra" }
 kompact = { git = "https://github.com/kompics/kompact.git", branch = "master"}
-uuid = "0.8"
+uuid = { version = "0.8", features = ["serde", "v4"] }
+static_assertions = "1.1.0"
+itertools = "0.8.2"
+smallbox = { version = "0.8", features = ["coerce"] }
 
 # Serialisation
 abomonation = "0.7.3"
 abomonation_derive = "0.5.0"
 prost = { git = "https://github.com/Max-Meldrum/prost.git", branch = "derive_generics" }
 prost-types = "0.6"
+serde = { version = "1.0.104", features = ["derive"] }
```

Comment:
Would it be possible to keep serde/bincode as a feature? Or is something very dependent on serde? e.g., serialisation of timers?

Reply:
I think I may be able to make everything use Prost, so serde would stay optional. That's one of the reasons I still have the PR as WIP 😄